spec/stat_tracker_spec.rb

```diff
@@ -22,8 +22,14 @@
     it 'has attributes' do
       expect(@stats.all_data.values.all? { |file| File.exist?(file) } ).to be true
     end
+
+    it 'makes using #create_teams, #create_games, and #create_game_teams' do
+      expect(@stats.instance_variable_get(:@teams)).to all be_a Team
```

Comment:
I like how this was tested; utilizes the instances that are created within the before(:each)

Reply:
It is definitely a neat way to do it.